CHANGELOG.md

```diff
@@ -6,232 +6,71 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Remove 'ADD_POPULAR_NETWORKS' feature flag as the feature is enabled by default ([#15229](https://github.com/MetaMask/metamask-extension/pull/15229))
+- Remove dark mode and transaction insights from the "What's New" announcements ([#15279](https://github.com/MetaMask/metamask-extension/pull/15279))
+- Update design tokens package and consolidate shadow values with new tokens ([#15264](https://github.com/MetaMask/metamask-extension/pull/15264))
+- Update Zendesk ticket request URLs for a facilitating the bug reporting process ([#15458](https://github.com/MetaMask/metamask-extension/pull/15458))
+- Update Customize Nonce 'Learn More' link pointing to the correct Zendesk article ([#15658](https://github.com/MetaMask/metamask-extension/pull/15658))
+- Update Avalanche RPC endpoint to point Infura, as it is now supported ([#15720](https://github.com/MetaMask/metamask-extension/pull/15720))
+- Update the main text on SetApprovalForAll confirmation screen ([#15724](https://github.com/MetaMask/metamask-extension/pull/15724))
+- Updated origin pill component to match the new design for Permissions and Switching Networks screens ([#15603](https://github.com/MetaMask/metamask-extension/pull/15603))
+- Set "View full transaction details" as default option on SetApprovalForAll Confirmation screen ([#15726](https://github.com/MetaMask/metamask-extension/pull/15726))
+- Update `gridplus-sdk` ([#15711](https://github.com/MetaMask/metamask-extension/pull/15711))
+  - `eth-lattice-keyring` changes: [GridPlus/eth-lattice-keyring@v0.11.0...v0.12.0](https://github.com/GridPlus/eth-lattice-keyring/compare/v0.11.0...v0.12.0)
+  - `gridplus-sdk` changes: [GridPlus/gridplus-sdk@v2.2.2...v2.2.7](https://github.com/GridPlus/gridplus-sdk/compare/v2.2.2...v2.2.7)
+- Update title case to sentence case with some exceptions (names, companies, special terms and page navigation) ([#15285](https://github.com/MetaMask/metamask-extension/pull/15285))
+- Swaps: show a block explorer URL and hide "Add token" prompt, if block explorer URL is not available on Build Quote screen ([#15198](https://github.com/MetaMask/metamask-extension/pull/15198))
+- Swaps: Standardize the spelling of 'cancelled' ([#15266](https://github.com/MetaMask/metamask-extension/pull/15266))
+- Swaps: Performance improvements for Swaps, especially on the Build Quote page (~7.2x faster) ([#15359](https://github.com/MetaMask/metamask-extension/pull/15359))
+- Swaps: only check if a user has enough balance before calling STX. If "Not enough funds" error is thrown, show a notification, but keep STX active ([#15218](https://github.com/MetaMask/metamask-extension/pull/15218))
+- [FLASK] Disable "Mark all as read button" when there are no notifications ([#15333](https://github.com/MetaMask/metamask-extension/pull/15333))
+- [FLASK] Update Snaps packages to version `^0.18.1` ([#15230](https://github.com/MetaMask/metamask-extension/pull/15230))
```

Comment:
I think this one can be omitted. Not much exciting user-facing happened in this update.

Reply:
@FrederikBolding do you confirm is it safe to remove the complete entry and sub-entries?

```
- [FLASK] Update Snaps packages to version `^0.18.1` ([#15230](https://github.com/MetaMask/metamask-extension/pull/15230))
  - Updates the following packages from `0.16.0` to `0.18.1`:
    - `@metamask/execution-environments`
    - `@metamask/rpc-methods`
    - `@metamask/snap-controllers`
    - `@metamask/snap-types`
  - Serialization issues were fixed in `@metmask/execution-environments`. See the ([releases](https://github.com/MetaMask/snaps-skunkworks/releases)) of the Snaps monorepo for details.
```

Reply:
Yeah. No real user-facing changes were introduced in this change. I also suggested that we change the other bump PRs to list the main user-facing features instead.

We could include every time we update the Snaps packages, but it doesn't seem that useful imo.

Reply:
Thanks @FrederikBolding I've removed the item and sub-items. Your suggestion sounds something we could talk on one of our Release meetings on Thursday. Feel free to jump in any week and we could further discuss it, if that sounds okay to you :)